prime-router/src/main/kotlin/cli/LivdTableExtracts.kt

```diff
@@ -0,0 +1,148 @@
+package gov.cdc.prime.router.cli
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.output.TermUi
+import com.github.ajalt.clikt.output.TermUi.prompt
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import org.apache.commons.io.FilenameUtils
+import org.apache.poi.hssf.usermodel.HSSFWorkbook
+import org.apache.poi.ss.usermodel.CellType
+import org.apache.poi.ss.usermodel.Row
+import org.apache.poi.ss.usermodel.Sheet
+import org.apache.poi.ss.usermodel.Workbook
+import org.apache.poi.xssf.usermodel.XSSFWorkbook
+import java.io.File
+import java.io.FileInputStream
+import java.io.FileOutputStream
+import java.io.IOException
+
+private const val sheetName = "LOINC Mapping"
+
+/**
+ * LivdTableExtracts is the commena line interface for the livd-table-extracts command. It parses the command line
+ * for option given as below.
+ */
+class LivdTableExtracks() : CliktCommand(
```

Comment:
This will not be a new command.  This needs to be part of the original download command, so this is a next step after the download.

Reply:
I will incorporate it into the LivdTableDownload command.

Reply:
Done.